utils/firestore.js

```diff
@@ -2270,16 +2280,22 @@ const addKitStatusToParticipant = async (participantsCID) => {
 const processParticipantHomeMouthwashKitData = (record, printLabel) => {
     const { collectionDetails, baseline, bioKitMouthwash, firstName, lastName, address1, address2, city, state, zip } = fieldMapping;
 
+    const addressLineOne = record?.[address1];
+    const poBoxRegex = /\b(?:P\.?O\.?(?:\s*Box|\s+Office\s+Box)|Post\s+Office\s+Box)\b/i;
+    const isPOBoxMatch = poBoxRegex.test(addressLineOne);
+    
+    if (isPOBoxMatch) return null;
+
     const hasMouthwash = record[collectionDetails][baseline][bioKitMouthwash] !== undefined;    
     const processedRecord = {
-        first_name: record[firstName],
-        last_name: record[lastName],
-        address_1: record[address1],
-        address_2: record[address2] || '',
-        city: record[city],
-        state: record[state],
-        zip_code: record[zip], 
-        connect_id: record['Connect_ID'],
+    first_name: record[firstName],
```

Comment:
Use JS conventions: no snake cases if you happen to adjust them.

Reply:
This is in snake case because it goes through the csv reports.